<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale spacer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>208</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>focal length (vert)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_mllsize">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>104</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size of the two MLLs in µm. This should be set sufficiently larger than the actual size.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>390</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Post height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>300</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Delta (opt const.)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>156</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coefficient (vert)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>330</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>µm</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>94</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>510</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>µm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>480</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Factor_Phase</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coefficient (horz)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>182</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>focal length (horz)</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>281</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_pillarwidth">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>360</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size of the stabilizing spacer of the fish.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>360</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>µm</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>238</y>
      <width>281</width>
      <height>38</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="Box_BB">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Specify if two stablizing boxes should be added to the model.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Bounding Box</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Box_P">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Specify if an underlying post should be added for easier alignment.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Post</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="LineEdit_f_v">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>208</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Focal length of the vertically focusing MLL (in µm).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>350</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_postheight">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>390</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size of the stabilizing spacer of the fish.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_pxsize">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>510</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The sampling size of the model (x,y).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_coeff_h">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Third order coefficient of the wavefront of the horizontally focusing MLL (in rad/mrad³).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>0.005</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>104</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Size of MLLs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>182</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>µm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>104</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>µm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_delta_mat">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>300</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Decrement of the refractive index of the used material at the given energy.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>1.72e-6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>390</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>µm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_N_lensletts">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>72</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of Lensletts. Each lenslett is fish-shaped and therefore has two refracting surfaces&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>156</y>
      <width>71</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>rad/mrad³</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Name">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Name of the model. Will create Name.stl and Name.log file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>CRC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_factor_phase">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>480</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;An additional factor to scale the magnitude of the correction.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_scale_spacer">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>330</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size of the stabilizing spacer of the fish.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Advanced:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>130</y>
      <width>71</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>rad/mrad³</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_coeff_v">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>156</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Third order coefficient of the wavefront of the vertically focusing MLL (in rad/mrad³).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>0.005</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>510</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pxsize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_f_h">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>182</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Focal length of the horizontally focusing MLL (in µm).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>250</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>360</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bounding Box size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>72</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>N Lensletts</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>208</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>µm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_SetDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Save Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPreviewMesh">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview Mesh</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="MPL_WIDGET" name="previewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>471</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>label_15</zorder>
   <zorder>RunButton</zorder>
   <zorder>label_9</zorder>
   <zorder>LineEdit_mllsize</zorder>
   <zorder>label_19</zorder>
   <zorder>label_12</zorder>
   <zorder>label_4</zorder>
   <zorder>label_16</zorder>
   <zorder>label_25</zorder>
   <zorder>label_23</zorder>
   <zorder>label_3</zorder>
   <zorder>label_7</zorder>
   <zorder>frame_3</zorder>
   <zorder>LineEdit_pillarwidth</zorder>
   <zorder>label_18</zorder>
   <zorder>frame_2</zorder>
   <zorder>LineEdit_f_v</zorder>
   <zorder>label_14</zorder>
   <zorder>LineEdit_postheight</zorder>
   <zorder>LineEdit_pxsize</zorder>
   <zorder>LineEdit_coeff_h</zorder>
   <zorder>label</zorder>
   <zorder>label_8</zorder>
   <zorder>label_2</zorder>
   <zorder>LineEdit_delta_mat</zorder>
   <zorder>label_20</zorder>
   <zorder>LineEdit_N_lensletts</zorder>
   <zorder>label_6</zorder>
   <zorder>LineEdit_Name</zorder>
   <zorder>LineEdit_factor_phase</zorder>
   <zorder>LineEdit_scale_spacer</zorder>
   <zorder>label_13</zorder>
   <zorder>label_5</zorder>
   <zorder>LineEdit_coeff_v</zorder>
   <zorder>label_24</zorder>
   <zorder>LineEdit_f_h</zorder>
   <zorder>label_17</zorder>
   <zorder>label_11</zorder>
   <zorder>label_10</zorder>
   <zorder>Button_SetDir</zorder>
   <zorder>ButtonPreviewMesh</zorder>
   <zorder>previewWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPL_WIDGET</class>
   <extends>QWidget</extends>
   <header location="global">mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
